search/search-client-elasticsearch/src/main/java/io/camunda/search/es/transformers/aggregator/SearchAggregationResultTransformer.java

```diff
@@ -70,10 +69,6 @@ private AggregationResult transformSingleBucketAggregate(
         .build();
   }
 
-  private AggregationResult transformLTermsBucketAggregate(final LongTermsAggregate aggregate) {
-    return new Builder().docCount((long) aggregate.buckets().array().size()).build();
```

Comment:
🔧 Looks like we're missing the doc count in some scenarios now, e.g., empty bucket arrays. We'll never set the doc count then. I'm not sure if the "global" doc count is even needed in any case, for example.